esp32s3-hal/examples/spi_slave_dma.rs

```diff
@@ -146,5 +146,59 @@ fn main() -> ! {
         );
 
         delay.delay_ms(250u32);
+
```

Comment:
Great you show the functionality in the example - would be nice to also update the examples for other supported chips to have the examples in-sync